### PR TITLE
Add MSC support for ESP32 S3

### PR DESCRIPTION
**Pull request by M-D-777**
Hi, all

Any interest in adding MSC support for ESP32 S3?

The MSC feature is configurable with sdkconfig.usb_msc.

### CODE REVIEW

File: ports/esp32/modules/inisetup.py [hunk 1]
```diff
@@ -33,8 +33,10 @@ def fs_corrupted():
 def setup():
     check_bootsec()
     print("Performing initial setup")
-    uos.VfsLfs2.mkfs(bdev)
-    vfs = uos.VfsLfs2(bdev)
+    # uos.VfsLfs2.mkfs(bdev)
```
Comment (PGNetHun) [inline ports/esp32/modules/inisetup.py:36]: Please do not change default `LittleFS` to `FAT`, as file is shared among all ESP32 variants (freeze in [manifest.py](https://github.com/micropython/micropython/blob/master/ports/esp32/boards/manifest.py#L1)).
Best way would be to have the `inisetup.py` file override in board folder with a custom `manifest.py`, just like in `UM_TINYS2` board's [manifest.py](https://github.com/micropython/micropython/blob/master/ports/esp32/boards/UM_TINYS2/manifest.py#L2) file

**Comment by Red-M** [comment]
I've tested this patch and it works perfectly, however I can these changes would affect other boards, please limit the changes to only the S3 boards.

**Comment by M-D-777** [comment]
@PGNetHun @Red-M Thanks for your review. I will update a commit later.

**Comment by M-D-777** [comment]
Hi, @PGNetHun I have added custom manifest.py for msc boot fatfs.
Please review the newest commit, thanks.

**Comment by M-D-777** [comment]
@Red-M @PGNetHun Hi, All. I have confirmed the USB MSC feature is compatible with ESP32 S2. So I add this feature to all of the ESP32 S2 and S3 variants.
By default, the usb feature is disabled by #boards/sdkconfig.usb_msc in mpconfigboard.cmake.
Do we need to enable the USB MSC feature by default, just like USB CDC?

**Pull request closed without merging**